pkg/streamingpromql/operators/functions/range_vectors.go

```diff
@@ -369,3 +369,56 @@ func avgHistograms(head, tail []promql.HPoint) (*histogram.FloatHistogram, error
 
 	return avgSoFar, nil
 }
+
+var Resets = FunctionOverRangeVectorDefinition{
+	SeriesMetadataFunction:         DropSeriesName,
+	StepFunc:                       resets,
+	NeedsSeriesNamesForAnnotations: true,
```

Comment:
Is this correct? I don't think `resets` emits any annotations, so this shouldn't be needed.

Reply:
I added this because some of failed instance test in `TestCompareVariousMixedMetricsVectorSelectors` due to annotation. But I think it failed because something else. Removed in 54d583988a7ce8befe1a631f188d05d1bd22d114.